src/lib/spinel/radio_spinel.hpp

```diff
@@ -69,10 +55,11 @@ class RadioSpinel : public SpinelInterface::Callbacks
     /**
      * Initialize this radio transceiver.
      *
-     * @param[in]  aPlatformConfig  Platform configuration structure.
+     * @param[in]  aResetRadio Whether to reset on init or not.
+     * @param[in]  aFetchNcp   Whether to support fetch data from NCP or not.
      *
      */
-    void Init(const otPlatformConfig &aPlatformConfig);
+    void Init(bool aResetRadio, bool aFetchNcp);
```

Comment:
Can we use the name "aRestoreDatasetFromNcp" ? It is consistent with the configuration name.